CharacterChatApi/appsettings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Host=localhost;Database=character_chat_api;Username=postgres;Password=postgres"
```

Comment:
Should we put the database credentials elsewhere? Seems like they shouldn't be included statically in this file.